htdocs/compta/bank/various_payment/card.php

```diff
@@ -48,9 +48,10 @@
 $accountid=GETPOST("accountid") > 0 ? GETPOST("accountid", "int") : 0;
 $label=GETPOST("label", "alpha");
 $sens=GETPOST("sens", "int");
-$amount=GETPOST("amount");
-$paymenttype=GETPOST("paymenttype");
+$amount=GETPOST("amount", "alpha");
+$paymenttype=GETPOST("paymenttype", "int");
```

Comment:
I think it should be "alpha" and not "int" (field in table is varchar), shouldn't it ?

Reply:
Hi @eldy ,

No, sorry, it's an int

![image](https://user-images.githubusercontent.com/2341395/52550412-58f2be00-2dd8-11e9-8f87-690a6edc5518.png)

![image](https://user-images.githubusercontent.com/2341395/52550440-72940580-2dd8-11e9-93a9-edf22955030c.png)